tf_publisher crashes when pressing RESET button in gazebo

When pressing the reset button in gazebo, the `tf_publisher.py` crashes with the following error message:

```
[ERROR] [WallTime: 1481794503.887025] [0.000000] ROS time moved backwards: 4.2s
[ERROR] [WallTime: 1481794503.887232] [0.000000] ROS time moved backwards: 4.204s
Traceback (most recent call last):
  File "/opt/ros/indigo/lib/cob_gazebo_worlds/scripts/tf_publisher.py", line 76, in <module>
    rate.sleep()
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/timer.py", line 99, in sleep
    sleep(self._remaining(curr_time))
  File "/opt/ros/indigo/lib/python2.7/dist-packages/rospy/timer.py", line 155, in sleep
    raise rospy.exceptions.ROSTimeMovedBackwardsException(time_jump)
rospy.exceptions.ROSTimeMovedBackwardsException: ROS time moved backwards
[ WARN] [1481794503.918072447, 0.020000000]: Detected jump back in time. Clearing TF buffer.
[world_tf_publisher-7] process has died [pid 7529, exit code 1, cmd /opt/ros/indigo/lib/cob_gazebo_worlds/scripts/tf_publisher.py __name:=world_tf_publisher __log:=/home/mig/.ros/log/b91de1de-c2a9-11e6-b453-901b0e08b6c2/world_tf_publisher-7.log].
```

--- a/cob_gazebo_worlds/scripts/tf_publisher.py
+++ b/cob_gazebo_worlds/scripts/tf_publisher.py
@@ -57,8 +57,6 @@
 #
 #################################################################
 
-import math
-
 import rospy
 import tf
 
@@ -67,10 +65,13 @@ if __name__ == '__main__':
     br = tf.TransformBroadcaster()
     rate = rospy.Rate(10.0)
     while not rospy.is_shutdown():
-        t = rospy.Time.now().to_sec()
         br.sendTransform((0,0,0.01),
                          (0.0, 0.0, 0.0, 1.0),
                          rospy.Time.now(),
                          "map",
                          "world")
-        rate.sleep()
+        
+        try:
+            rate.sleep()
+        except rospy.ROSTimeMovedBackwardsException, e:
+            rospy.logwarn("ROSTimeMovedBackwardsException during sleep(). Continue anyway...")

